CRM-12874

dlobo opened the pull request:
Renamed it to civicrm.settings.php.template

CRM-12874: Rename civicrm.settings.php.tpl so it isn't in the same suffix as Smarty templates http://issues.civicrm.org/jira/browse/CRM-12874

Commit:
CRM-12874

----------------------------------------
* CRM-12874: Move civicrm.settings.php.tpl so it isn't in the same place as Smarty templates
  http://issues.civicrm.org/jira/browse/CRM-12874

totten commented:
I think the drush/provision process tries to support multiple versions. Might do:

``` php
$files = array(
  "$crmPath/templates/CRM/common/civicrm.settings.php.template",
  "$crmPath/templates/CRM/common/civicrm.settings.php.tpl"
);
foreach ($files as $file) {
  if (file_exists($file)) {
    $settingsTplFile = $file;
  }
}
```

ping @mlutfy

Commit:
CRM-12874

----------------------------------------
* CRM-12874: Rename civicrm.settings.php.tpl so it uses a different suffix from Smarty templates
  http://issues.civicrm.org/jira/browse/CRM-12874

dlobo commented:
fixed and recommitted
